crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -1403,6 +1406,20 @@ impl CumulocityConverter {
         &mut self,
         message: &MqttMessage,
     ) -> Result<Vec<MqttMessage>, ConversionError> {
+        if self.active_commands_last_cleared.elapsed() > Duration::from_secs(3600) {
```

Comment:
I was actually thinking that you'd use the timer actor from the mapper to get timeout notification message at desired intervals. But, this was smart.